configs/test-study.nrel-op.json

```diff
@@ -0,0 +1,80 @@
+{
+    "version": 1,
+    "ts": 1655143472,
+    "server": {
+        "connectUrl": "https://openpath-test.nrel.gov/api/",
```

Comment:
I would suggest having this follow the same pattern as the other deployments (e.g. `deployment-openpath.nrel.gov`). We created the stage environment before coming up with the pattern, but that meant that we have a bunch of special cases for stage in various places.

Let's not add to that with this new environment.

Reply:
```suggestion
        "connectUrl": "https://test-openpath.nrel.gov/api/",
```

Reply:
Updated